docs/sections/release_notes/getting-started.md

```diff
@@ -0,0 +1,242 @@
+---
+id: getting-started
```

Comment:
two things:
- this id is the same as the other page in the tutorial section
- is getting-started.md an appropriate name for this page?